src/docfx/build/document/DocumentProvider.cs

```diff
@@ -62,6 +62,17 @@ public string GetOutputPath(FilePath path)
             return UrlUtility.Combine(_config.BasePath, outputPath);
         }
 
+        public string GetDocsSiteUrl(FilePath path, OutputUrlType outputUrlType)
```

Comment:
If this already takes `OutputUrlType`, consider remove `Docs` from the name
```suggestion
        public string GetSiteUrl(FilePath path, OutputUrlType outputUrlType)
```

Reply:
The parameter `outputUrlType` means the current config outputUrlType, not the target urlType

---
In reply to: [415203921](https://github.com/dotnet/docfx/pull/5826#discussion_r415203921) [](ancestors = 415203921)